src/server/pps/server/poller.go

```diff
@@ -50,20 +50,19 @@ func (m *ppsMaster) cancelPipelinePodsPoller() {
 	}
 }
 
-// startPipelineEtcdPoller starts a new goroutine running
-// pollPipelinesEtcd
-func (m *ppsMaster) startPipelineEtcdPoller() {
+// startPipelineDBPoller starts a new goroutine running watchPipelines
+func (m *ppsMaster) startPipelineWatcher() {
```

Comment:
Scrubbed mentions of etcd from this file (because it's postgres now), and did a slight renaming pass on stuff that got touched (this poller wasn't actually polling, it was watching).